variables.tf

```diff
@@ -1,5 +1,45 @@
-
 variable "aws_account_id" {
   type        = string
   description = "The id of the aws account we want to extract the metadata from."
 }
+
+variable "stage" {
+  type        = string
+  description = "Stage of the outputs, e.g. dev, tst, acc, prd"
+}
+
+variable "deployment" {
+  type        = string
+  default     = "main"
+  description = "Deployment of the resource, e.g. main, b, c, d"
+}
+
+variable "network" {
+  description = "Return outputs from network baseline"
+  default     = {
+    enable     = true
+    stage      = null
+    deployment = null
+  }
+}
+
+variable "ecs" {
+  description = "Return outputs from ECS baseline"
+  default     = {
+    enable     = true
+    stage      = null
+    deployment = null
+  }
+}
+
+variable "cloudmap" {
+  description = "Return outputs from CloudMap baseline"
+  default     = {
+    enable     = true
+    stage      = null
+    deployment = null
+  }
+}
+
```

Comment:
as there is a risk some SSM parameters are not defined for specific services as cloudmap or ecs, maybe we can set the enable to `false` , so the user should enable only what he needs, only that should **exists** . In that way we avoid any logic to deal with unexistent parameters. What do you think guys ?

Reply:
I agree. The next question then is if all will be false by default and users will set it to true, or if we control what is true/false by default based on which baselines are deployed. First option seems more consistent to me.